e2e/nodes/test_summarizer.py

```diff
@@ -42,26 +37,40 @@
 ]
 
 
-@pytest.mark.integration
-@pytest.mark.summarizer
+@pytest.fixture
+def summarizer():
+    return TransformersSummarizer(model_name_or_path="sshleifer/distilbart-xsum-12-6", use_gpu=False)
+
+
+@pytest.fixture
+def retriever(request):
+    if request.param == "bm25":
+        retriever = BM25Retriever(document_store=ElasticsearchDocumentStore())
```

Comment:
We might use InMemory for these tests as well, now that it supports BM25

Reply:
I'd prefer not to get into test refactoring at this stage, just moving